rdmd.d

```diff
@@ -845,19 +832,17 @@ string makeEvalFile(string todo)
     auto srcfile = buildPath(pathname,
             "eval." ~ todo.md5Of.toHexString ~ ".d");
 
-    if (force || !exists(srcfile))
+    if (force || !Yap.exists(srcfile))
     {
         std.file.write(srcfile, todo);
```

Comment:
Why not via the wrapper?

Reply:
this log statement was logged before, I didn't see any reason to add it

Reply:
Note, that just because a function is in `std.file` doesn't mean it needs to be logged.

Reply:
Why not?

Reply:
I personally don't see logging this as helpful to understand what rdmd is doing.  If you think it's useful I can add logging for it.  However, it looks like you are basing "logging usefulness" on if a function is in `std.file`.  Note that this isn't necessarily the case.  If we want to add logging here, I suggest we determine why we think it would be useful, and then go through the rest of the file and use the same criteria to add logging everywhere else as well.  However, that should probably be in it's own PR anyway.  This PR is only attempting to use existing criteria to determine when to log, it just adds some missing statements and makes it harder to forget them in the future.

Reply:
Actually, now that I think about it, knowing that `rdmd` is writing to a file would be useful to know.  I'll go ahead and add this one to the wrapper.

Reply:
> However, it looks like you are basing "logging usefulness" on if a function is in std.file. 

It's not really that; it's more that I think that not being thorough in isolating all write access to the filesystem dilutes the point of this PR.

It's easier to see how "now ALL access to the filesystem goes through the wrapper, and we are enforcing this by removing the top-level imports" is easier to argue about than "we created a wrapper and made some filesystem operations go through it, but we haven't really been consistent about which ones do and which ones don't, and using the wrapper is opt-in so you can just pretend it doesn't exist if you want to".

Reply:
Same goes for consistency of logging; if the goal of the PR is to improve logging, then it has a stronger case if it goes all the way than just halway.

Reply:
Ok, I've moved pretty much everything to the wrapper...

Reply:
Yeah, I suppose the criteria of "performing an operation on the file system" on knowing when to log makes sense.  And that just so happens to coincide with the `std.file` module.